variables.tf

```diff
@@ -15,6 +15,14 @@ variable "apiary_tags" {
   type        = map(any)
 }
 
+variable "apiary_extra_tags" {
```

Comment:
can we rename this to `apiary_extra_tags_s3`

Reply:
I'll set it as apiary_extra_tags_s3_logs, is that ok ?

Reply:
I noticed it was more a generic one with any extra tag, so I left extra_tags_s3